spring/src/main/java/com/alibaba/fescar/spring/util/SpringProxyUtils.java

```diff
@@ -48,10 +48,31 @@
             Object target = advised.getTargetSource().getTarget();
             return findTargetClass(target);
         } else {
+            if(proxy == null){
+                return null;
+            }
             return proxy.getClass();
         }
     }
 
+    public static Class<?>[] findInterfaces(Object proxy) throws Exception {
+        if(AopUtils.isJdkDynamicProxy(proxy)){
```

Comment:
If it's a CGLIB proxy class, I don't think it's necessary to consider whether the interface it implements needs to be proxyed, otherwise we should refactor it, scan all normal classes, its parent class, and the interfaces it implements.

Reply:
The reason why we need to scan the jdk proxy is that there are many such implementations : 
only the interface exists, there is no specific implementation class, the user can only mark the implementation of the fescar interception on the interface.